### DISCUSSION TITLE
db 업뎃 언제 끝남??

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#925] [질문]
끝나면 알려주세요잉

번역 추가 할거있음둥

**Comment by RimWorldKorea** [comment]
완료

**Reply by RimWorldKorea** [reply]
올릴거 올리면 스팀에도 걍 올려버려
지금 이슈 남아있는것들 오늘 밤늦게나 내일 작업할 예정이라

**Reply by zilrty** [reply]
올리는거 아마 내일 하지싶음

낚시하러 나와서 언제 집갈지 몰?루 ㅋㅋ
